authorization/scc/tc538263/PodSecurityPolicySubjectReview.json

```diff
@@ -0,0 +1,60 @@
+{
+    "kind": "PodSecurityPolicySubjectReview",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "pspsr"
+    },
+    "spec": {
+        "template": {
+            "spec": {
+            "containers": [
+                {
+                    "name": "hello-openshift",
+                    "image": "aosqe/hello-openshift",
+                    "ports": [
+                        {
+                            "containerPort": 8080,
+                            "protocol": "TCP"
+                        }
+                    ],
+                    "resources": {
+
+                    },
+                    "volumeMounts": [
+                        {
+                            "name": "tmp",
+                            "mountPath": "/tmp"
+                        }
+                    ],
+                    "terminationMessagePath": "/dev/termination-log",
+                    "imagePullPolicy": "IfNotPresent",
+                    "securityContext": {
+                        "capabilities": {
+
+                        },
+                        "privileged": false
+                    }
+                }
+            ],
+            "volumes": [
+                {
+                    "name": "tmp",
+                    "emptyDir": {
+
+                    }
+                }
+            ],
+            "restartPolicy": "Always",
+            "dnsPolicy": "ClusterFirst",
+            "serviceAccountName": "default"
+            }
+        },
+        "user": "wjiang",
```

Comment:
Don't care about it, when the file downloaded and will be replaceed by the real namesapce name.